x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/store/KibanaOwnedReservedRoleDescriptors.java

```diff
@@ -90,7 +90,7 @@ static RoleDescriptor kibanaSystem(String name) {
             new RoleDescriptor.IndicesPrivileges[] {
                 // System indices defined in KibanaPlugin
                 RoleDescriptor.IndicesPrivileges.builder()
-                    .indices(".kibana*", ".reporting-*")
+                    .indices(".kibana*", ".reporting-*", ".kibana-reporting*")
```

Comment:
@dakrone doesn't existing `".kibana*"` pattern cover `".kibana-reporting*"` already?

Reply:
Yes, it does. I thought it might help to be explicit, but if not I'm happy to remove it (the `.kibana-reporting` name is still pending, so it may end up not starting with `.kibana`).

Reply:
I'm happy with the `.kibana-reporting` name

Reply:
> I thought it might help to be explicit,

I feel like it adds more confusion than value. I spent a minute or so trying to understand why `kibana*` wouldn't match `.kibana-reporting*` 🙈. We specifically use `.kibana*` to avoid modifying this role descriptor for every Kibana internal index

Reply:
Sounds good, I've removed it now.